.circleci/set-up-globals.sh

```diff
@@ -17,21 +17,7 @@ apt-get update
 composer global require -n "hirak/prestissimo:^0.3"
 
 # Install Terminus
-if [ ! -d $HOME/terminus ]
-then
-	# Clone terminus if it doesn't exist
-	echo -e "Installing Terminus...\n"
-	git clone --branch master git://github.com/pantheon-systems/terminus.git $HOME/terminus
-	cd $HOME/terminus
-	composer install
-	cd -
-else
-	# Otherwise make sure terminus is up to date
-	cd $HOME/terminus
-	git pull
-	composer install
-	cd -
-fi
+composer global require pantheon-systems/terminus
```

Comment:
@greg-1-anderson this was giving me `terminus: command not found` even though it is added to `$PATH`. See [build 346](https://circleci.com/gh/pantheon-systems/example-wordpress-composer/346)